Automattic/jetpack
# Add "Connections" to the "Hosting" menu

Related to https://github.com/Automattic/dotcom-forge/issues/5766

## Proposed changes:


* Adds "Connections" to the "Hosting" menu

<img width="829" alt="Screenshot 2024-03-08 at 4 24 34 PM" src="https://github.com/Automattic/jetpack/assets/140841/3df7aad9-e779-4f66-89df-ac7b074e73aa">


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* To setup follow the instructions in the comment below: https://github.com/Automattic/jetpack/pull/36302#issuecomment-1986496611.
* Observe the link exists on an Atomic classic site.
* Click the link as see that it goes to https://wpcalypso.wordpress.com/marketing/connections/[site_slug]
